examples/keras_io/tensorflow/vision/cct.py

```diff
@@ -214,9 +255,9 @@ def __init__(self, drop_prop, **kwargs):
     def call(self, x, training=None):
         if training:
             keep_prob = 1 - self.drop_prob
-            shape = (tf.shape(x)[0],) + (1,) * (tf.shape(x).shape[0] - 1)
-            random_tensor = keep_prob + tf.random.uniform(shape, 0, 1)
-            random_tensor = tf.floor(random_tensor)
+            shape = (keras.ops.shape(x)[0],) + (1,) * (len(x.shape) - 1)
```

Comment:
`(tf.shape(x).shape[0] - 1)` made Torch fail (error message was complaining about too many dimensions to the tensor). I believe it was a typo from the original implementation, which I replaced by `(len(x.shape) - 1)`